ui/app/components/totp-edit.hbs

```diff
@@ -26,17 +26,18 @@
 </PageHeader>
 
 {{#if (eq @mode "show")}}
-  <Totp::KeyDetails @model={{@model}} @delete={{this.delete}} />
+  <Totp::KeyDetails @model={{@model}} @delete={{this.deleteKey}} />
```

Comment:
it's helpful to name component args (like we did below) so it's clear what the passed item is. Like if this was a boolean I'd expect the arg to be `@isDeleted` but this is a function so I suggest renaming to
```suggestion
  <Totp::KeyDetails @model={{@model}} @onDelete={{this.deleteKey}} />
```